opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -65,18 +65,20 @@ namespace Opm {
         m_nz = static_cast<size_t>( ecl_grid_get_nz( c_ptr() ));
     }
 
-    EclipseGrid::EclipseGrid(const ecl_grid_type * src_ptr)
+    /* Copy constructor */
+    EclipseGrid::EclipseGrid(const EclipseGrid& src)
         : m_minpvValue(0),
```

Comment:
> Should not all these member values be copied from src?

Is there any best practice in a situation like this; should I access the members of src directly - or through their accessors?